tests/testsuite/cargo_new/inherit_workspace_package_table_without_version/mod.rs

```diff
@@ -0,0 +1,22 @@
+use cargo_test_support::compare::assert_ui;
+use cargo_test_support::curr_dir;
+use cargo_test_support::CargoCommand;
+use cargo_test_support::Project;
+
+#[cargo_test]
+fn case() {
+    let project = Project::from_template(curr_dir!().join("in"));
+    let project_root = project.root();
+    let cwd = &project_root;
+
+    snapbox::cmd::Command::cargo_ui()
+        .arg("new")
+        .args(["crates/foo", "--edition", "2021"])
```

Comment:
How come we use `--edition` in this test?

Reply:
Removed.